### PR TITLE
I dont know what ive done, but it works on my machine

### PR DESCRIPTION
**Pull request by Noksit**

### CODE REVIEW

**Comment by Noksit** [comment]
WHY THE HELL MY PULL REQUEST HAS BEEN CLOSE ????

**Comment by mbaechler** [comment]
@MelBou , @Tabardelli : you are supposed to review this PR for tomorrow morning

**Comment by MelBou** [comment]
TestCountry.java → nice use of the country list to choose the right CSV file. 
 Tests seem alright. I never used tuple, pretty interesting method !

Country.java → nothing to say, getters/setters/toString. All good.

Main.java → why did you chose to write your code in the main file ? 
Be careful with your documentation, for example :  on line 56, you’re saying the method’s gonna return « List<City> » but it actually returns the companyList. 
Same thing happens on lines 114 and 150. 

Obviously, you’re in advance on the job they asked us to do, so you made way more things than expected by Matthieu Baechler. Everything seems alright and interesting though. 
Well done guys !

File: java/java-entreprise-map.iml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mbaechler) [inline java/java-entreprise-map.iml]: please remove iml files from your commit

Comment (mbaechler) [reply java/java-entreprise-map.iml]: and .class files too

**Comment by mbaechler** [comment]
Good work

**Pull request closed without merging**